### ISSUE TITLE
Chillers reporting sizing to eio multiple times

### ISSUE THREAD

**Issue by Myoldmopar** [NREL/EnergyPlus#4656]
The chillers are reporting their autosized (and user-sized) values to the eio multiple times.  This should be protected.  Many example files are showing it.  For example, ```ElectricChiller```'s eio file has:

```
$ sort eplusout.eio | uniq -d --count
     11  Component Sizing Information, Chiller:Electric, BIG CHILLER, User-Specified Design Chilled Water Flow Rate [m3/s], 1.10000E-003
     11  Component Sizing Information, Chiller:Electric, BIG CHILLER, User-Specified Design Condenser Water Flow Rate [m3/s], 1.10000E-003
     11  Component Sizing Information, Chiller:Electric, BIG CHILLER, User-Specified Nominal Capacity [W], 100000.00000
      2 Environment:Daylight Saving,No,SizingPeriod:DesignDay
      2 Environment:WarmupDays,  6
 ```

indicating those chiller sizing lines are repeated verbatim 11 times.

**Comment by mjwitte** [comment]
@Myoldmopar This is still happening as of 8.2.9, upping the priority.

**Comment by EnergyArchmage** [comment]
This is another major bug that was fixed in new plant code for HVAC Sizing Simulation

**Comment by Myoldmopar** [comment]
Fixed in #4764.
